jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/steps/NavigationRunSteps.java

```diff
@@ -43,7 +43,7 @@ public void expandGroupInOperationsTab(final String groupName) {
         // check first if the group is already expanded, if it is there is no need to expand it
         if (jwalaUi.isElementExists(
                 By.xpath("//td[text()='" + groupName + "']/preceding-sibling::td//span[contains(@class,'ui-icon-triangle-1-e')]"))) {
-            jwalaUi.click(By.xpath("//td[text()='" + groupName + "']/preceding-sibling::td"));
+            jwalaUi.click(By.xpath("//td[text()='" + groupName + "']/preceding-sibling::td//span"));
```

Comment:
Isn't this a different search than the previous XPath? Do all td elements contain spans for this test?

Reply:
I have enabled native click for IE driver. So, we have to provide exact tag to make click function work. In this function, We must have to click expand button with certain properties "//span[contains(@class,'ui-icon-triangle-1-e')]" to expand the section. It was not functioning with new capabilities provided. So, i have added a "//span" in xpath to provide exact locator for driver to perform the operation.